packages/expo-calendar/CHANGELOG.md

```diff
@@ -11,6 +11,7 @@
 ### 🐛 Bug fixes
 
 - Fix `Calendar.getEventsAsync` crashing when `recurrenceRules` are incorrect. ([#8760](https://github.com/expo/expo/pull/8760) by [@lukmccall](https://github.com/lukmccall))
+- Fixed `stringifyDate` helper functions. ([#9269](https://github.com/expo/expo/pull/9269) by [@barthap](https://github.com/barthap))
```

Comment:
This changelog entry would be good and clear for us internally, but not really for our users as the `stringifyDate` is not being exported by the module and it's not a part of our API. Could you rephrase it to be more about user-facing changes (an issue that this PR will close might be helpful)?

Reply:
I think this should be better now
https://github.com/expo/expo/pull/9269/files#diff-75d1dd6f04ae24a7dfb0e11eb467376a